File: core/src/main/java/com/uiptv/model/SeriesWatchingNowSnapshot.java
package com.uiptv.model;

import com.uiptv.shared.BaseJson;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@EqualsAndHashCode(callSuper = false)
public class SeriesWatchingNowSnapshot extends BaseJson {
    private String dbId;
    private String accountId;
    private String categoryId;
    private String seriesId;
    private String categoryDbId;
    private String seriesTitle;
    private String seriesPoster;
    private String episodesJson;
    private long updatedAt;
}

File: server/src/main/kotlin/com/uiptv/model/SeriesWatchingNowSnapshot.kt
package com.uiptv.model

import com.uiptv.shared.BaseJson

data class SeriesWatchingNowSnapshot @JvmOverloads constructor(
    var dbId: String? = null,
    var accountId: String? = null,
    var categoryId: String? = null,
    var seriesId: String? = null,
    var categoryDbId: String? = null,
    var seriesTitle: String? = null,
    var seriesPoster: String? = null,
    var episodesJson: String? = null,
    var updatedAt: Long = 0
) : BaseJson()
